2017-10-13T16:30:35.031425+00:00 heroku[web.1]: Stopping all processes with SIGTERM
2017-10-13T16:30:35.048091+00:00 app[web.1]: [2017-10-13 16:30:35 +0000] [4] [INFO] Handling signal: term
2017-10-13T16:30:35.048510+00:00 app[web.1]: [2017-10-13 16:30:35 +0000] [9] [INFO] Worker exiting (pid: 9)
2017-10-13T16:30:35.050913+00:00 app[web.1]: [2017-10-13 16:30:35 +0000] [8] [INFO] Worker exiting (pid: 8)
2017-10-13T16:30:36.984227+00:00 app[web.1]: [2017-10-13 16:30:36 +0000] [4] [INFO] Shutting down: Master
2017-10-13T16:30:37.164959+00:00 heroku[web.1]: Process exited with status 0
2017-10-13T18:43:07.000000+00:00 app[api]: Build started by user jessica.zheng.2015@gmail.com
2017-10-13T18:43:54.972907+00:00 heroku[web.1]: State changed from down to starting
2017-10-13T18:43:54.757053+00:00 app[api]: Deploy 9abaaa68 by user jessica.zheng.2015@gmail.com
2017-10-13T18:43:07.000000+00:00 app[api]: Build succeeded
2017-10-13T18:43:54.757053+00:00 app[api]: Release v34 created by user jessica.zheng.2015@gmail.com
2017-10-13T18:44:08.716895+00:00 heroku[web.1]: Starting process with command `gunicorn app:app`
2017-10-13T18:44:12.626221+00:00 heroku[web.1]: Process exited with status 1
2017-10-13T18:44:12.633142+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:12.633165+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 209, in run
2017-10-13T18:44:12.633173+00:00 app[web.1]:     self.sleep()
2017-10-13T18:44:12.633180+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 359, in sleep
2017-10-13T18:44:12.633181+00:00 app[web.1]:     ready = select.select([self.PIPE[0]], [], [], 1.0)
2017-10-13T18:44:12.633182+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 244, in handle_chld
2017-10-13T18:44:12.633183+00:00 app[web.1]:     self.reap_workers()
2017-10-13T18:44:12.633183+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 524, in reap_workers
2017-10-13T18:44:12.098322+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [4] [INFO] Starting gunicorn 19.7.1
2017-10-13T18:44:12.099465+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [4] [INFO] Listening at: http://0.0.0.0:57808 (4)
2017-10-13T18:44:12.099582+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [4] [INFO] Using worker: sync
2017-10-13T18:44:12.104949+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [8] [INFO] Booting worker with pid: 8
2017-10-13T18:44:12.115724+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [8] [ERROR] Missing parentheses in call to 'print' (app.py, line 97)
2017-10-13T18:44:12.115728+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:12.115729+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
2017-10-13T18:44:12.633184+00:00 app[web.1]:     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
2017-10-13T18:44:12.115730+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2017-10-13T18:44:12.115731+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
2017-10-13T18:44:12.115731+00:00 app[web.1]:     self.callable = self.load()
2017-10-13T18:44:12.115732+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2017-10-13T18:44:12.115733+00:00 app[web.1]:     return self.load_wsgiapp()
2017-10-13T18:44:12.115733+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2017-10-13T18:44:12.633194+00:00 app[web.1]: gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
2017-10-13T18:44:12.115734+00:00 app[web.1]:     return util.import_app(self.app_uri)
2017-10-13T18:44:12.115735+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/util.py", line 352, in import_app
2017-10-13T18:44:12.115735+00:00 app[web.1]:     __import__(module)
2017-10-13T18:44:12.633203+00:00 app[web.1]:
2017-10-13T18:44:12.115736+00:00 app[web.1]:   File "/app/app.py", line 97
2017-10-13T18:44:12.115737+00:00 app[web.1]:     print 'subreddit_of_interest updated to ', subreddit_of_interest
2017-10-13T18:44:12.115737+00:00 app[web.1]:                                             ^
2017-10-13T18:44:12.115743+00:00 app[web.1]: SyntaxError: Missing parentheses in call to 'print'
2017-10-13T18:44:12.117054+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [8] [ERROR] Exception in worker process
2017-10-13T18:44:12.117056+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:12.633204+00:00 app[web.1]: During handling of the above exception, another exception occurred:
2017-10-13T18:44:12.117057+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
2017-10-13T18:44:12.117058+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2017-10-13T18:44:12.117059+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
2017-10-13T18:44:12.117059+00:00 app[web.1]:     self.callable = self.load()
2017-10-13T18:44:12.117060+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2017-10-13T18:44:12.633205+00:00 app[web.1]:
2017-10-13T18:44:12.117061+00:00 app[web.1]:     return self.load_wsgiapp()
2017-10-13T18:44:12.117062+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2017-10-13T18:44:12.117063+00:00 app[web.1]:     return util.import_app(self.app_uri)
2017-10-13T18:44:12.117063+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/util.py", line 352, in import_app
2017-10-13T18:44:12.117064+00:00 app[web.1]:     __import__(module)
2017-10-13T18:44:12.117065+00:00 app[web.1]:   File "/app/app.py", line 97
2017-10-13T18:44:12.633205+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:12.117065+00:00 app[web.1]:     print 'subreddit_of_interest updated to ', subreddit_of_interest
2017-10-13T18:44:12.117066+00:00 app[web.1]:                                             ^
2017-10-13T18:44:12.117067+00:00 app[web.1]: SyntaxError: Missing parentheses in call to 'print'
2017-10-13T18:44:12.117067+00:00 app[web.1]:
2017-10-13T18:44:12.117068+00:00 app[web.1]: During handling of the above exception, another exception occurred:
2017-10-13T18:44:12.117068+00:00 app[web.1]:
2017-10-13T18:44:12.633206+00:00 app[web.1]:   File "/app/.heroku/python/bin/gunicorn", line 11, in <module>
2017-10-13T18:44:12.117069+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:12.633207+00:00 app[web.1]:     sys.exit(run())
2017-10-13T18:44:12.117070+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
2017-10-13T18:44:12.117070+00:00 app[web.1]:     worker.init_process()
2017-10-13T18:44:12.633208+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 74, in run
2017-10-13T18:44:12.117071+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 126, in init_process
2017-10-13T18:44:12.117072+00:00 app[web.1]:     self.load_wsgi()
2017-10-13T18:44:12.117072+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 148, in load_wsgi
2017-10-13T18:44:12.117073+00:00 app[web.1]:     self.reloader.add_extra_file(exc_val.filename)
2017-10-13T18:44:12.117077+00:00 app[web.1]: AttributeError: 'NoneType' object has no attribute 'add_extra_file'
2017-10-13T18:44:12.117241+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [8] [INFO] Worker exiting (pid: 8)
2017-10-13T18:44:12.151518+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [9] [INFO] Booting worker with pid: 9
2017-10-13T18:44:12.207582+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [9] [ERROR] Missing parentheses in call to 'print' (app.py, line 97)
2017-10-13T18:44:12.207586+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:12.207587+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
2017-10-13T18:44:12.207588+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2017-10-13T18:44:12.207589+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
2017-10-13T18:44:12.207590+00:00 app[web.1]:     self.callable = self.load()
2017-10-13T18:44:12.207590+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2017-10-13T18:44:12.633208+00:00 app[web.1]:     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
2017-10-13T18:44:12.207591+00:00 app[web.1]:     return self.load_wsgiapp()
2017-10-13T18:44:12.207591+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2017-10-13T18:44:12.207592+00:00 app[web.1]:     return util.import_app(self.app_uri)
2017-10-13T18:44:12.207592+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/util.py", line 352, in import_app
2017-10-13T18:44:12.633209+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 203, in run
2017-10-13T18:44:12.207593+00:00 app[web.1]:     __import__(module)
2017-10-13T18:44:12.207596+00:00 app[web.1]:   File "/app/app.py", line 97
2017-10-13T18:44:12.207597+00:00 app[web.1]:     print 'subreddit_of_interest updated to ', subreddit_of_interest
2017-10-13T18:44:12.207597+00:00 app[web.1]:                                             ^
2017-10-13T18:44:12.633210+00:00 app[web.1]:     super(Application, self).run()
2017-10-13T18:44:12.207603+00:00 app[web.1]: SyntaxError: Missing parentheses in call to 'print'
2017-10-13T18:44:12.208881+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [9] [ERROR] Exception in worker process
2017-10-13T18:44:12.208883+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:12.208884+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
2017-10-13T18:44:12.208884+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2017-10-13T18:44:12.633211+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 72, in run
2017-10-13T18:44:12.208885+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
2017-10-13T18:44:12.208886+00:00 app[web.1]:     self.callable = self.load()
2017-10-13T18:44:12.208886+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2017-10-13T18:44:12.208887+00:00 app[web.1]:     return self.load_wsgiapp()
2017-10-13T18:44:12.208887+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2017-10-13T18:44:12.208888+00:00 app[web.1]:     return util.import_app(self.app_uri)
2017-10-13T18:44:12.208889+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/util.py", line 352, in import_app
2017-10-13T18:44:12.208890+00:00 app[web.1]:     __import__(module)
2017-10-13T18:44:12.633212+00:00 app[web.1]:     Arbiter(self).run()
2017-10-13T18:44:12.208890+00:00 app[web.1]:   File "/app/app.py", line 97
2017-10-13T18:44:12.208891+00:00 app[web.1]:     print 'subreddit_of_interest updated to ', subreddit_of_interest
2017-10-13T18:44:12.208891+00:00 app[web.1]:                                             ^
2017-10-13T18:44:12.208892+00:00 app[web.1]: SyntaxError: Missing parentheses in call to 'print'
2017-10-13T18:44:12.208895+00:00 app[web.1]:
2017-10-13T18:44:12.208896+00:00 app[web.1]: During handling of the above exception, another exception occurred:
2017-10-13T18:44:12.633213+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 231, in run
2017-10-13T18:44:12.208896+00:00 app[web.1]:
2017-10-13T18:44:12.208897+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:12.208897+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
2017-10-13T18:44:12.208902+00:00 app[web.1]:     worker.init_process()
2017-10-13T18:44:12.208903+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 126, in init_process
2017-10-13T18:44:12.208904+00:00 app[web.1]:     self.load_wsgi()
2017-10-13T18:44:12.208904+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 148, in load_wsgi
2017-10-13T18:44:12.633213+00:00 app[web.1]:     self.halt(reason=inst.reason, exit_status=inst.exit_status)
2017-10-13T18:44:12.208905+00:00 app[web.1]:     self.reloader.add_extra_file(exc_val.filename)
2017-10-13T18:44:12.208910+00:00 app[web.1]: AttributeError: 'NoneType' object has no attribute 'add_extra_file'
2017-10-13T18:44:12.209079+00:00 app[web.1]: [2017-10-13 18:44:12 +0000] [9] [INFO] Worker exiting (pid: 9)
2017-10-13T18:44:12.633214+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 344, in halt
2017-10-13T18:44:12.633215+00:00 app[web.1]:     self.stop()
2017-10-13T18:44:12.633216+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 393, in stop
2017-10-13T18:44:12.633216+00:00 app[web.1]:     time.sleep(0.1)
2017-10-13T18:44:12.633217+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 244, in handle_chld
2017-10-13T18:44:12.633218+00:00 app[web.1]:     self.reap_workers()
2017-10-13T18:44:12.633219+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 524, in reap_workers
2017-10-13T18:44:12.633219+00:00 app[web.1]:     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
2017-10-13T18:44:12.633220+00:00 app[web.1]: gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
2017-10-13T18:44:12.638107+00:00 heroku[web.1]: State changed from starting to crashed
2017-10-13T18:44:12.639952+00:00 heroku[web.1]: State changed from crashed to starting
2017-10-13T18:44:24.548856+00:00 heroku[web.1]: Starting process with command `gunicorn app:app`
2017-10-13T18:44:26.828669+00:00 app[web.1]: [2017-10-13 18:44:26 +0000] [4] [INFO] Starting gunicorn 19.7.1
2017-10-13T18:44:26.829231+00:00 app[web.1]: [2017-10-13 18:44:26 +0000] [4] [INFO] Listening at: http://0.0.0.0:49788 (4)
2017-10-13T18:44:26.829306+00:00 app[web.1]: [2017-10-13 18:44:26 +0000] [4] [INFO] Using worker: sync
2017-10-13T18:44:26.832532+00:00 app[web.1]: [2017-10-13 18:44:26 +0000] [8] [INFO] Booting worker with pid: 8
2017-10-13T18:44:26.836761+00:00 app[web.1]: [2017-10-13 18:44:26 +0000] [8] [ERROR] Missing parentheses in call to 'print' (app.py, line 97)
2017-10-13T18:44:26.836762+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:26.836763+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
2017-10-13T18:44:26.836763+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2017-10-13T18:44:26.836764+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
2017-10-13T18:44:26.836764+00:00 app[web.1]:     self.callable = self.load()
2017-10-13T18:44:26.836765+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2017-10-13T18:44:26.836765+00:00 app[web.1]:     return self.load_wsgiapp()
2017-10-13T18:44:26.836766+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2017-10-13T18:44:26.836766+00:00 app[web.1]:     return util.import_app(self.app_uri)
2017-10-13T18:44:26.836767+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/util.py", line 352, in import_app
2017-10-13T18:44:26.836767+00:00 app[web.1]:     __import__(module)
2017-10-13T18:44:26.836767+00:00 app[web.1]:   File "/app/app.py", line 97
2017-10-13T18:44:26.836768+00:00 app[web.1]:     print 'subreddit_of_interest updated to ', subreddit_of_interest
2017-10-13T18:44:26.836768+00:00 app[web.1]:                                             ^
2017-10-13T18:44:26.836770+00:00 app[web.1]: SyntaxError: Missing parentheses in call to 'print'
2017-10-13T18:44:26.837386+00:00 app[web.1]: [2017-10-13 18:44:26 +0000] [8] [ERROR] Exception in worker process
2017-10-13T18:44:26.837387+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:26.837387+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
2017-10-13T18:44:26.837388+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2017-10-13T18:44:26.837388+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
2017-10-13T18:44:26.837388+00:00 app[web.1]:     self.callable = self.load()
2017-10-13T18:44:26.837389+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
2017-10-13T18:44:26.837389+00:00 app[web.1]:     return self.load_wsgiapp()
2017-10-13T18:44:26.837390+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
2017-10-13T18:44:26.837390+00:00 app[web.1]:     return util.import_app(self.app_uri)
2017-10-13T18:44:26.837391+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/util.py", line 352, in import_app
2017-10-13T18:44:26.837391+00:00 app[web.1]:     __import__(module)
2017-10-13T18:44:26.837391+00:00 app[web.1]:   File "/app/app.py", line 97
2017-10-13T18:44:26.837392+00:00 app[web.1]:     print 'subreddit_of_interest updated to ', subreddit_of_interest
2017-10-13T18:44:26.837392+00:00 app[web.1]:                                             ^
2017-10-13T18:44:26.837392+00:00 app[web.1]: SyntaxError: Missing parentheses in call to 'print'
2017-10-13T18:44:26.837393+00:00 app[web.1]:
2017-10-13T18:44:26.837393+00:00 app[web.1]: During handling of the above exception, another exception occurred:
2017-10-13T18:44:26.837394+00:00 app[web.1]:
2017-10-13T18:44:26.837394+00:00 app[web.1]: Traceback (most recent call last):
2017-10-13T18:44:26.837394+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
2017-10-13T18:44:26.837395+00:00 app[web.1]:     worker.init_process()
2017-10-13T18:44:26.837395+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 126, in init_process
2017-10-13T18:44:26.837395+00:00 app[web.1]:     self.load_wsgi()
2017-10-13T18:44:26.837396+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/gunicorn/workers/base.py", line 148, in load_wsgi
2017-10-13T18:44:26.837396+00:00 app[web.1]:     self.reloader.add_extra_file(exc_val.filename)
2017-10-13T18:44:26.837412+00:00 app[web.1]: AttributeError: 'NoneType' object has no attribute 'add_extra_file'
2017-10-13T18:44:26.837528+00:00 app[web.1]: [2017-10-13 18:44:26 +0000] [8] [INFO] Worker exiting (pid: 8)
2017-10-13T18:44:26.864419+00:00 app[web.1]: [2017-10-13 18:44:26 +0000] [4] [INFO] Shutting down: Master
2017-10-13T18:44:26.864533+00:00 app[web.1]: [2017-10-13 18:44:26 +0000] [4] [INFO] Reason: Worker failed to boot.
2017-10-13T18:44:26.949947+00:00 heroku[web.1]: Process exited with status 3
2017-10-13T18:44:26.964075+00:00 heroku[web.1]: State changed from starting to crashed
2017-10-13T18:44:28.651858+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=reddit-visualizer-iw03.herokuapp.com request_id=84517760-b5fd-4d6a-9393-c9209a9d6d8f fwd="140.180.242.25" dyno= connect= service= status=503 bytes= protocol=https
